modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/serializer/SchedulerSerializer.java

```diff
@@ -175,7 +177,9 @@ private static ScheduleCalendar createCalendar( final PropertySet data )
                 return calendar.build();
             }
 
-            return null; // TODO: return OneTimeCalendar
+            return OneTimeCalendar.create().
```

Comment:
That leaves no room for future improvements. We need to store calendar type in another filed, I guess.

Reply:
Also, higher up: seems like it is "possible" to write cron without timezone - and that becomes ambiguous